index.php

```diff
@@ -25,7 +26,7 @@
 			</ul>
 			<div class="title">
 				<h1>Kyam Harris</h1>
-				<h4>Somerset based Web Developer</h4>
+				<h4><?php echo $abouts[0]['content']?></h4>
```

Comment:
if you always only need the first item of an array try `fetch()` rather than `fetchAll()`

Reply:
Variable is used elsewhere